packages/base/src/models/model.ts

```diff
@@ -568,6 +574,14 @@ export class QueryGridModel extends Record({
         }
     }
 
+    static getEmptySelection() : any {
```

Comment:
Is there a reason this is a method and not just a constant declared at the top of the file (or in an equivalent `constants.ts` file)?

Reply:
No good reason, no.  Just didn't think of it. :-)

Reply:
Actually, I think I considered it, but was avoiding exporting the constant as a global thing since it is so much tied to the model, so I've compromised (I hope) and put it as a static member of the class rather than using a getter.